src/providers/hana/qgshanautils.cpp

```diff
@@ -417,6 +417,20 @@ QVersionNumber QgsHanaUtils::toHANAVersion( const QString &dbVersion )
   return QVersionNumber( maj, min, rev );
 }
 
+QVersionNumber QgsHanaUtils::toHANACloudVersion( const QString &dbCloudVersion )
```

Comment:
This function is almost identical to the one above. I don't see any value of having two functions. We can combine them in the following way:

```
QVersionNumber QgsHanaUtils::toHANAVersion( const QString &dbVersion )
{
  QString version = dbVersion;
  QStringList strs = version.replace( '-', '.' ).replace( ' ', '.' ).split( '.' );

  if ( strs.length() < 3 )
    return QVersionNumber( 0 );

  const int maj = strs[0].toInt();
  const int min = strs[1].toInt();
  const int rev = strs[2].toInt();
  return QVersionNumber( maj, min, rev );
}
```

Reply:
Done.